### PR TITLE
feat: add abbreviation tooltips

### PR DESCRIPTION
**Pull request by Sleinmaster**
## Summary
- Adds tooltips for abbreviations
- Make abbreviations visible in explanations as discussed in https://github.com/flybywiresim/docs/pull/771#discussion_r1146753749

Discord username (if different from GitHub): Slein#8982

### CODE REVIEW

**Comment by Valastiri** [comment]
This feature isn't in insiders? (If not awesome lol)

File: mkdocs.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ theme:
   palette:
     scheme: slate
   features:
+    - content.tooltips
```
Comment (Sleinmaster) [inline mkdocs.yml]:
```suggestion
```
Doesn't work without insiders

**Comment by Valastiri** [comment]
check build

**Comment by Sleinmaster** [comment]
I think if there are no wishes for additional general abbreviations, we can merge the pr as is, might add a few more often used abbreviations.
And then we'll have to watch on the pages which additional abbreviations would be useful as tooltip.

**Comment by Valastiri** [comment]
Will take a look at this tonight

**Pull request merged**